addon/templates/components/cfb-form-editor/question.hbs

```diff
@@ -3,12 +3,9 @@
     {{uk-spinner ratio=2}}
   </div>
 {{else}}
-  {{#with model as |model|}}
+  {{#with model}}
```

Comment:
This looked strange before, and now looks even stranger :smile: Shall we replace it with a simple `{{if}}`?